libs/components/src/next/form/select/vl-select.component.ts

```diff
@@ -0,0 +1,309 @@
+                                        ${
+                                            this.multiple
+                                                ? `<span class="vl-pill__close__icon" aria-hidden="true"></span>`
+                                                : ''
+                                        }
+                                    </button>
+                                </div>`
+                            );
+                        }
+
+                        return template(`
+                            <div class="vl-select__item
+                                ${data.highlighted ? 'is-highlighted' : 'vl-select__item--selectable'}
+                                ${this.multiple ? 'vl-pill' : ''}
+                                ${data.placeholder ? 'vl-select__placeholder' : ''}"
+                                data-item
+                                data-id="${data.id}"
+                                data-value="${data.value}"
+                                ${data.disabled ? 'aria-disabled="true"' : ''}
+                            >
+                                ${data.label}
+                            </div>
+                        `);
+                    },
+
+                    itemList: () => {
+                        if (!this.multiple) {
+                            return template(`<div class="vl-input-field"></div>`);
+                        }
+
+                        return template(`<div class="vl-input-field vl-select__list--multiple"></div>`);
+                    },
+                    input: () => {
+                        return template(
+                            `<input type="text" class="vl-input-field vl-input-field-cloned" autocomplete="off" autocapitalize="off" spellcheck="false" role="textbox" aria-autocomplete="list" aria-label="zoek item">`
+                        );
+                    },
+                    dropdown: () => {
+                        return template(
+                            `<div class="vl-select__list vl-select__list--dropdown" role="group" id="vl-select__list"></div>`
+                        );
+                    },
+                    choiceList: () => {
+                        return template(
+                            `<div class="vl-select__list" role="listbox" aria-label="item lijst" tabindex="0"></div>`
+                        );
+                    },
+                };
+            },
+        };
+    }
+
+    private getOptions(): SelectOption[] {
+        const options = [...this.options];
+
+        if (this.placeholder && !this.multiple) {
+            const placeholderOption: SelectOption = {
+                value: '',
+                label: this.placeholder,
+                placeholder: true,
+                disabled: true,
+                selected: true,
+            };
+            options.unshift(placeholderOption);
+        }
+
+        return options;
+    }
+
+    private onChange(): void {
```

Comment:
`void` hoeft niet expliciet vermeld te worden

Reply:
zoals besproken, wordt dit dan eventueel later opgenomen en gaan we ook duidelijk `code style` definiëren